x/liquidstaking/types/insurance.go

```diff
@@ -55,8 +55,8 @@ func SortInsurances(validatorMap map[string]stakingtypes.Validator, insurances [
 		iInsurance := insurances[i]
 		jInsurance := insurances[j]
 
-		iValidator := validatorMap[iInsurance.ValidatorAddress]
-		jValidator := validatorMap[jInsurance.ValidatorAddress]
+		iValidator := validatorMap[iInsurance.GetValidator().String()]
+		jValidator := validatorMap[jInsurance.GetValidator().String()]
```

Comment:
Is there a reason why you had to change it like this?

Reply:
No special reason for this. 
I just thought `GetValidator().String()` is more uniform than `insurance.ValidatorAddress` because when create validatorMap in `GetNetAmountStateEssentials`, it uses `Validator().String()` as a key. 

But, I saw that both `GetValidator().String()` and `insurance.ValidatorAddress` are used in liquidstaking module's code.
Both values are same, so it would be great to unify that usage in code base for readability.

Reply:
[`GetValidator()`](https://github.com/Canto-Network/Canto/blob/b6c34839eb021a68a55ab7eb936a88465145ab3f/x/liquidstaking/types/insurance.go#L43-L46) eventually computes using `insurance.ValidatorAddress`, and already has the resulting string in the `insurance.ValidatorAddress`,  it doesn't seem necessary to compute it again

Reply:
@dongsam 
Oh.. I see why u comment for it. 
I got it! Thanks for the detail.

I'll check other `GetValidator` usages too and update that. Thanks!

Reply:
@dongsam 
updated it! https://github.com/b-harvest/Canto/pull/39/commits/8b66c5c4ec114e8d6dd189dda6cc4fff39efa6b9 

the reason why tests failed yesterday is that I changed https://github.com/b-harvest/Canto/pull/39/commits/8b66c5c4ec114e8d6dd189dda6cc4fff39efa6b9#diff-ab2f09d261b7e9f72f62052a1d86a36ce8cff3b911bb79243a8d7c603c866d9bR1609 to `!=` yesterday. It must be equal, so the logic was broken. But now it is fixed (using `==`).